Liber Usualis - mei3/liberupdatev5.py

```diff
@@ -0,0 +1,239 @@
+            child.tag = "peeb"
+            for att in child.attrib:                
+                if att.endswith("pageref"):                    
+                    pageref = child.attrib.pop(att)
+                    break
+            for child_2 in root.iter():
+              for attr, value in child_2.attrib.items():               
+                if attr.endswith("id") and value == pageref:
+                        child.attrib["n"] = child_2.attrib["n"]
+                        break                 
+            pa_bre = ET.tostring(child)
+            theres_a_pb = True
+            child.tag = "TODELETE"
+        
+        # Handle system breaks
+        if child.tag.endswith("sb"):
+            child.tag = "seeb"
+            for att in child.attrib:
+                if att.endswith("systemref"):                    
+                    systemref = child.attrib.pop(att)
+                    break
+            for child_2 in root.iter():               
+              for attr, value in child_2.attrib.items():
+                if attr.endswith("id") and value == systemref:
+                        child.attrib["facs"] = child_2.attrib["facs"]
+                        break
+            sy_bre = ET.tostring(child)           
+            theres_a_sb = True
+            child.tag = "TODELETE"
+
+        # Moves system breaks and page breaks inside layer
+        if child.tag.endswith("layer"):
+            if theres_a_pb:
+                theres_a_pb = False
+                child.append(ET.fromstring(pa_bre))
+            if theres_a_sb:
+                theres_a_sb = False
+                child.append(ET.fromstring(sy_bre))
+
+        # Handle zone elements (adjusts negative ulx and lrx attributes)
+        if child.tag.endswith("zone"):
+             label = ""
+             for attr, value in child.attrib.items():
+                if attr.endswith("ulx") and int(value) < 0:                       
+                        child.attrib["label"] = label + "ulx = " + value + " "
+                        child.attrib[attr] = str(0)
+                        break                
+             for attr, value in child.attrib.items():
+                if attr.endswith("lrx") and int(value) < 0:       
+                        child.attrib["label"] = label + "lrx = " + value + " "
+                        child.attrib[attr] = str(0)
+                        break
+        
+        # Handle dot elements (temporary fix until morae get added to MEI)
+        if child.tag.endswith("dot"):
+            child.tag = "signifLet"
+            for att in child.attrib:
+                if att.endswith("form"):
+                    child.attrib.pop(att)
+                    break
+
+        # Handle neume elements (become syllables, ncs become neumes, and notes become ncs)
+        if child.tag.endswith("neume"):
+            syl = ET.SubElement(child, "syl")
```

Comment:
`syl` is not used

Reply:
Going by https://docs.python.org/3/library/xml.etree.elementtree.html#building-xml-documents this is the proper syntax for adding subelements, even when the variables do not get used later

Reply:
So:
1. Why are you creating a bunch of empty `syl` elements?
2. If you want to assign it to a variable name, give it a name that signals that this is intentional. `_` is often used by convention, `unused_syl` is fine too.